### DISCUSSION TITLE
请问支持数字跳跃那种动画吗？并支持替换最新数字

### DISCUSSION THREAD

**Discussion by FaDeve** [Tencent/libpag#132] [Ideas] [answered]
请问支持数字跳跃那种动画吗？并支持替换最新数字

**Comment by kevingpqi123** [comment, marked as answer]
在AE中使用PAG支持的矢量特性做出来的动画，文本和图片都是支持可编辑的
如需要数字跳动类动画，如从00顺序跳动到99之间任意一个数字，设计一个完整的从00到99的动画，然后控制播放进度
